icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -6,19 +6,21 @@
 #if !UCONFIG_NO_FORMATTING
 
 #include "charstr.h"
-#include <cstdarg>
-#include <cmath>
-#include <memory>
-#include "unicode/unum.h"
-#include "unicode/numberformatter.h"
-#include "unicode/testlog.h"
-#include "unicode/utypes.h"
 #include "number_asformat.h"
+#include "number_microprops.h"
 #include "number_types.h"
 #include "number_utils.h"
 #include "number_utypes.h"
-#include "number_microprops.h"
 #include "numbertest.h"
+#include "unicode/displayoptions.h"
```

Comment:
While you are re-sorting header files... Best practice for the order of includes is
- C standard headers
- C++ standard headers
- ICU public headers (the ones with unicode/)
- ICU internal headers

Reply:
done